sys/sys/linker.h

```diff
@@ -32,6 +32,7 @@
 #ifdef _KERNEL
 
 #include <machine/elf.h>
+#include <sys/systm.h>
```

Comment:
This is extreme namespace pollution.
I suspect it is due to preload_initkmdp(), and I do not see why it needs to be inline.